ui/articles-aem-renderer/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">ui/articles-aem-renderer</string>
+</resources>
```

Comment:
this needs to go away so it doesn't interfere with the app_name set elsewhere :)